### PR TITLE
[TW-1225] Split importing and exporting

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Split "Importing and exporting data" into four separate topics in a subfolder
- Added content to new overview topic
- Updated left nav
- Updated links and redirects

### CODE REVIEW

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/getting-started/importing-and-exporting/exporting-data.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+* [Exporting environments](#exporting-environments)
+* [Exporting data dumps](#exporting-data-dumps)
+
+## Exporting collections
+
+1. Select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to the collection, then select **Export**.
+
+   <img alt="Export collection" src="https://assets.postman.com/postman-docs/export-collection-v9.1.jpg" width="350px" />
+
+1. Select the format you'd like your collection to export as.
```
Comment (nerminamiller-postman) [inline src/pages/docs/getting-started/importing-and-exporting/exporting-data.md:28]:
```suggestion
1. Select the export format for your collection.
```

File: src/pages/docs/getting-started/importing-and-exporting/importing-data.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+You can import Postman data you exported earlier, including collections, environments, data dumps, and globals.
+
+1. Select **Import** in the left navigation menu.
+1. Do one of the following:
+    * Select your file or folder, or drag it into the **Import** modal.
+    * Paste your raw text or URL into the input field.
+    * Select a button to import from a [GitHub](/docs/getting-started/importing-and-exporting/importing-from-git/#importing-from-github-repositories), [Bitbucket](/docs/getting-started/importing-and-exporting/importing-from-git/#importing-from-bitbucket-repositories), or [GitLab](/docs/getting-started/importing-and-exporting/importing-from-git/#importing-from-gitlab-repositories) code repository.
+    * Select **Other Sources** to import an [Azure](/docs/getting-started/importing-and-exporting/importing-from-git/#importing-from-azure-devops-repositories) code repository, or to [import an API from Amazon API Gateway or Azure API Management](/docs/designing-and-developing-your-api/importing-an-api/).
+    Postman will automatically recognize Postman data, confirming the name, format, and what the file will import as.
```
Comment (nerminamiller-postman) [inline src/pages/docs/getting-started/importing-and-exporting/importing-data.md:67]:
```suggestion
    Postman will automatically recognize Postman data, confirming the name, format, and type.
```

File: src/pages/docs/getting-started/importing-and-exporting/importing-data.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+To convert your collection's format from v1 to v2, do the following:
+
+1. In the terminal of your choice, enter the following command to install the Postman Collection Transformer.
+
+   ```bash
+   sudo npm install -g postman-collection-transformer
+   ```
+
+1. Convert an individual Postman Collection from v1 to v2 by entering this command:
```
Comment (nerminamiller-postman) [inline src/pages/docs/getting-started/importing-and-exporting/importing-data.md:87]:
```suggestion
1. Convert an individual Postman collection from v1 to v2 by entering this command:
```

File: src/pages/docs/getting-started/importing-and-exporting/importing-data.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+   ```bash
+   postman-collection-transformer convert -i <path to the input Postman Collection file> -o <path to the downloaded Postman file> -j 1.0.0 -p 2.0.0 -P
+   ```
+
+The resulting collection will download to your target file path in v2 format.
+
+You can retrieve a list of convert options by running the command with the ``-h`` flag:
```
Comment (nerminamiller-postman) [inline src/pages/docs/getting-started/importing-and-exporting/importing-data.md:95]:
```suggestion
You can retrieve a list of convert options by running the command with the `-h` flag:
```

**Review by nerminamiller-postman — APPROVED** [review]
LGTM! Very nice! I left just a few small suggestions.

File: redirects.json [hunk 5]
```diff
@@ -669,23 +669,23 @@
   },
   {
     "from": "/docs/postman/collections/data-formats/",
-    "to": "/docs/getting-started/importing-and-exporting-data/"
+    "to": "/docs/getting-started/importing-and-exporting/importing-and-exporting-overview/"
```
[the remark this answers is not in the document]
Comment (jkonrath-postman) [reply redirects.json:672]: Hold off on switching that for now.

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**